internals/secrethub/audit.go

```diff
@@ -104,9 +102,6 @@ func (cmd *AuditCommand) run() error {
 	header := strings.Join(auditTable.header(), "\t") + "\n"
 	fmt.Fprint(tabWriter, header)
 
-	// interactive mode is assumed, except when output is piped.
-	interactive := !cmd.io.Stdout().IsPiped() && !cmd.noPrompt
-
 	i := 0
 	for {
 		i++
```

Comment:
Should we be defensive here and make this an `uint64` to prevent overflows? Technically the loop can run indefinitely.

Reply:
`i` is reset to 0 when it reaches the number of items per page (line 127)
The number of items per page is checked to be positive (and not zero) (line 57)

We could make both of them `uint` to enforce this in the types as well :+1: